package.json

```diff
@@ -42,13 +42,44 @@
 		"prepush": "npm run lint && npm test"
 	},
 	"devDependencies": {
+		"@types/chai": "4.1.5",
+		"@types/expect": "1.20.3",
+		"@types/jquery": "3.3.9",
+		"@types/mocha": "5.2.5",
+		"@types/node": "10.10.1",
```

Comment:
Also include "types/chai-as-promised":"7.1.0", "types/sinon":"5.0.5", and "types/sinon-chai":"3.2.0"

Reply:
Also, we need `"@types/bip39": "2.4.0"` for passphrase

Reply:
Are these already exists in `development` branch?

Reply:
Not yet but in near future PRs for typescript. Add now or later?

Reply:
I think it make sense to add them when it requires.
When we the new PR are merged, anyway we need to clean up the `package.json` anyway.

Also, if the another PR goes in first, then we can do opposite